projects/ad9213_dual_ebz/common/ad9213_dual_qsys.tcl

```diff
@@ -29,71 +33,49 @@ set_instance_parameter_value ad9213_rx_1 {SYSCLK_FREQUENCY} {100.0}
 set_instance_parameter_value ad9213_rx_1 {REFCLK_FREQUENCY} {320.3125}
 set_instance_parameter_value ad9213_rx_1 {INPUT_PIPELINE_STAGES} {2}
 set_instance_parameter_value ad9213_rx_1 {NUM_OF_LANES} {16}
+set_instance_parameter_value ad9213_rx_1 {EXT_DEVICE_CLK_EN} {1}
+
+# Link merger
+
+add_instance ad9213_dual_link adi_jesd204_link_merge
+set_instance_parameter_value ad9213_dual_link {NUM_OF_LANES_PER_LINK} {16}
 
 # ad9213_tpl_0 JESD204B transport layer
 
-add_instance axi_ad9213_0 ad_ip_jesd204_tpl_adc
-set_instance_parameter_value axi_ad9213_0 {ID} {0}
-set_instance_parameter_value axi_ad9213_0 {NUM_CHANNELS} {2}
-set_instance_parameter_value axi_ad9213_0 {NUM_LANES} {16}
-set_instance_parameter_value axi_ad9213_0 {BITS_PER_SAMPLE} {16}
-set_instance_parameter_value axi_ad9213_0 {CONVERTER_RESOLUTION} {16}
-set_instance_parameter_value axi_ad9213_0 {TWOS_COMPLEMENT} {1}
-
-add_instance axi_ad9213_1 ad_ip_jesd204_tpl_adc
-set_instance_parameter_value axi_ad9213_1 {ID} {1}
-set_instance_parameter_value axi_ad9213_1 {NUM_CHANNELS} {2}
-set_instance_parameter_value axi_ad9213_1 {NUM_LANES} {16}
-set_instance_parameter_value axi_ad9213_1 {BITS_PER_SAMPLE} {16}
-set_instance_parameter_value axi_ad9213_1 {CONVERTER_RESOLUTION} {16}
-set_instance_parameter_value axi_ad9213_1 {TWOS_COMPLEMENT} {1}
+add_instance axi_ad9213_dual ad_ip_jesd204_tpl_adc
+set_instance_parameter_value axi_ad9213_dual {ID} {0}
+set_instance_parameter_value axi_ad9213_dual {NUM_CHANNELS} {2}
```

Comment:
I think the number of channels must double just as the number of lanes.

Reply:
Good point. Although the ADC has one single physical converter, it can support M=2 as two virtual converters. I've contacted the BU to tell us what would be the best default configuration. As I'll get an answer will update this. Thanks!

Reply:
The default configuration of the ADC will have M=1, so we can leave NUM_CHANNELS to be set to 2. Thanks for pointing this out anyhow.